### DISCUSSION TITLE
ampache 5.1.1 - Subsonic: No folders

### DISCUSSION THREAD

**Discussion by bacinet02** [ampache/ampache#3109] [Q&A] [answered]
Hello.

I have a new installation of ampache.
The connexion with subsonic client is ok.
In subsonic client, I see and can select catalogs.
But I see any album!
In webclient, I see them.

Can you help me?
Thanks for answers :-)

**Comment by bacinet02** [comment]
Problem is solved:
. I have installed and use DSub (android)
. To find folders, I had to login for the 1st time on the web
. Problem still exists with subsonic application (android)

:-)

**Reply by lachlan-00** [reply]
interesting! i didn't know that. maybe the mapping wasn't updated correctly

working on updating the queries for the artist lists too so that might help

**Reply by Rodol66** [reply]
Could it be related to the problem I reported at https://github.com/ampache/ampache/discussions/3096?

I have continued doing tests with Dsub and the problem with the folders that do not appear if I do not activate the browsing by tags appears only in the list of artists, if you search for any other concept (genre, recently added, even random) they appear correctly.

Greetings

**Reply by lachlan-00** [reply]
yes, i think there are issues with artist grouping that i've just fixed with 2679275f3 and some earlier query fixes should help fill that data now.

**Reply by Rodol66** [reply]
the problem continues in 5.1.2, nothing appears searching by artist

**Reply by lachlan-00** [reply]
is your catalog_map table still missing artists?

**Comment by Rodol66** [comment, marked as answer]
I have reviewed it and it was because of the Dsub cache, it works perfectly.  Very good job, thank you very muchEl 14 feb 2022 7:16, lachlan ***@***.***> escribió:
is your catalog_map table still missing artists?
